### ISSUE TITLE
Content with required tag field cannot be published

### ISSUE THREAD

**Issue by dsolistorres** [dotCMS/core#16026]
Reported via support ticket: [97470](https://dotcms.zendesk.com/agent/tickets/97470)
Reproducible in dotCMS 5.0.3

Content from content type having a required tag field cannot be published. An error message is raised: "Error: Contentlet has invalid / missing field(s)"

## Expected Behavior
Content items from a content type with a required field of tag type should be published if the field is valid (it has some value set for tag). 

## Current Behavior
When trying to publish a content item in working state from the Content Search portlet, a error message is shown even if the field has a value set for the tag (the tag field is required for the content type). The error message is: "Error: Contentlet has invalid / missing field(s)"

## Possible Solution
Fix the code to include the tag values for the field in the content map used in the validation, so the validation code don't evaluate the field as empty and trigger the error signaling the the value is missing from the field.

## Known Workarounds
Edit the content item and click Save/Publish to publish it, instead of choosing Publish from the Content Search portlet context menu.

## Steps to Reproduce (for bugs)
1. Log into dotCMS 5.0.3
2. Create a new content type (or edit an existing one)
3. Add a field of tag type and set it as required
4. Go to Content Search portlet, and create a new content item of the content type, choose a not empty value for the tag field, and save it (without publishing)
5. Right click the content item, choose Publish action.
6. An error message is shown stating that a field is missing for the content:
<img width="921" alt="invalid-field" src="https://user-images.githubusercontent.com/2894221/52655675-56689500-2eba-11e9-82a2-23360cf66b81.png">

The following error message is the dotcms.log file:
```
[12/02/19 11:15:32:131 CST] ERROR business.WorkflowAPIImpl: There was an unexpected error: Contentlet [53be627a-9864-4803-8cbd-56349adb8197] has invalid / missing field(s).
[12/02/19 11:15:32:132 CST] ERROR ajax.BrowserAjax: Contentlet [53be627a-9864-4803-8cbd-56349adb8197] has invalid / missing field(s).
com.dotmarketing.portlets.workflows.business.DotWorkflowException: Contentlet [53be627a-9864-4803-8cbd-56349adb8197] has invalid / missing field(s).
	at com.dotmarketing.portlets.workflows.business.WorkflowAPIImpl.fireWorkflowPostCheckin_aroundBody148(WorkflowAPIImpl.java:2025) ~[dotcms_5.0.0_999999.jar:?]
	at com.dotmarketing.portlets.workflows.business.WorkflowAPIImpl$AjcClosure149.run(WorkflowAPIImpl.java:1) ~[dotcms_5.0.0_999999.jar:?]
	at org.aspectj.runtime.reflect.JoinPointImpl.proceed(JoinPointImpl.java:149) ~[aspectjrt-1.8.10.jar:?]
	at com.dotcms.aspects.aspectj.AspectJDelegateMethodInvocation.proceed(AspectJDelegateMethodInvocation.java:43) ~[dotcms_5.0.0_999999.jar:?]
	at com.dotmarketing.db.LocalTransaction.wrapReturnWithListeners(LocalTransaction.java:40) ~[dotcms_5.0.0_999999.jar:?]
	at com.dotcms.aspects.interceptors.WrapInTransactionMethodInterceptor.invoke(WrapInTransactionMethodInterceptor.java:23) ~[dotcms_5.0.0_999999.jar:?]
	at com.dotcms.aspects.aspectj.WrapInTransactionAspect.invoke(WrapInTransactionAspect.java:41) ~[dotcms_5.0.0_999999.jar:?]
	at com.dotmarketing.portlets.workflows.business.WorkflowAPIImpl.fireWorkflowPostCheckin(WorkflowAPIImpl.java:1980) ~[dotcms_5.0.0_999999.jar:?]
	at com.dotmarketing.portlets.workflows.business.WorkflowAPIImpl.fireWorkflowNoCheckin_aroundBody168(WorkflowAPIImpl.java:2842) ~[dotcms_5.0.0_999999.jar:?]
	at com.dotmarketing.portlets.workflows.business.WorkflowAPIImpl$AjcClosure169.run(WorkflowAPIImpl.java:1) ~[dotcms_5.0.0_999999.jar:?]
	at org.aspectj.runtime.reflect.JoinPointImpl.proceed(JoinPointImpl.java:149) ~[aspectjrt-1.8.10.jar:?]
	at com.dotcms.aspects.aspectj.AspectJDelegateMethodInvocation.proceed(AspectJDelegateMethodInvocation.java:43) ~[dotcms_5.0.0_999999.jar:?]
	at com.dotmarketing.db.LocalTransaction.wrapReturnWithListeners(LocalTransaction.java:40) ~[dotcms_5.0.0_999999.jar:?]
	at com.dotcms.aspects.interceptors.WrapInTransactionMethodInterceptor.invoke(WrapInTransactionMethodInterceptor.java:23) ~[dotcms_5.0.0_999999.jar:?]
	at com.dotcms.aspects.aspectj.WrapInTransactionAspect.invoke(WrapInTransactionAspect.java:41) ~[dotcms_5.0.0_999999.jar:?]
	at com.dotmarketing.portlets.workflows.business.WorkflowAPIImpl.fireWorkflowNoCheckin(WorkflowAPIImpl.java:2838) ~[dotcms_5.0.0_999999.jar:?]
	at com.dotmarketing.portlets.browser.ajax.BrowserAjax.saveFileAction(BrowserAjax.java:544) ~[dotcms_5.0.0_999999.jar:?]
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:1.8.0_181]
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62) ~[?:1.8.0_181]
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:1.8.0_181]
	at java.lang.reflect.Method.invoke(Method.java:498) ~[?:1.8.0_181]
	at com.dotcms.repackage.org.directwebremoting.impl.CreatorModule$1.doFilter(CreatorModule.java:229) ~[dot.dwr-3rc2modified_3.jar:?]
	at com.dotcms.repackage.org.directwebremoting.impl.CreatorModule.executeMethod(CreatorModule.java:241) ~[dot.dwr-3rc2modified_3.jar:?]
	at com.dotcms.repackage.org.directwebremoting.impl.DefaultRemoter.execute(DefaultRemoter.java:379) ~[dot.dwr-3rc2modified_3.jar:?]
	at com.dotcms.repackage.org.directwebremoting.impl.DefaultRemoter.execute(DefaultRemoter.java:332) ~[dot.dwr-3rc2modified_3.jar:?]
	at com.dotcms.repackage.org.directwebremoting.dwrp.BaseCallHandler.handle(BaseCallHandler.java:104) ~[dot.dwr-3rc2modified_3.jar:?]
	at com.dotcms.repackage.org.directwebremoting.servlet.UrlProcessor.handle(UrlProcessor.java:120) ~[dot.dwr-3rc2modified_3.jar:?]
	at com.dotcms.repackage.org.directwebremoting.servlet.DwrServlet.doPost(DwrServlet.java:141) ~[dot.dwr-3rc2modified_3.jar:?]
	at com.dotmarketing.servlets.DwrWrapperServlet.doPost(DwrWrapperServlet.java:60) ~[dotcms_5.0.0_999999.jar:?]
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:661) ~[servlet-api.jar:?]
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:742) ~[servlet-api.jar:?]
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:231) ~[catalina.jar:8.5.32]
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166) ~[catalina.jar:8.5.32]
	at org.apache.tomcat.websocket.server.WsFilter.doFilter(WsFilter.java:52) ~[tomcat-websocket.jar:8.5.32]
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:193) ~[catalina.jar:8.5.32]
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166) ~[catalina.jar:8.5.32]
	at com.dotmarketing.filters.CMSFilter.doFilterInternal(CMSFilter.java:199) ~[dotcms_5.0.0_999999.jar:?]
	at com.dotmarketing.filters.CMSFilter.doFilter(CMSFilter.java:56) ~[dotcms_5.0.0_999999.jar:?]
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:193) ~[catalina.jar:8.5.32]
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166) ~[catalina.jar:8.5.32]
	at com.dotcms.filters.interceptor.AbstractWebInterceptorSupportFilter.doFilter(AbstractWebInterceptorSupportFilter.java:90) ~[dotcms_5.0.0_999999.jar:?]
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:193) ~[catalina.jar:8.5.32]
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166) ~[catalina.jar:8.5.32]
	at com.dotcms.filters.interceptor.AbstractWebInterceptorSupportFilter.doFilter(AbstractWebInterceptorSupportFilter.java:90) ~[dotcms_5.0.0_999999.jar:?]
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:193) ~[catalina.jar:8.5.32]
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166) ~[catalina.jar:8.5.32]
	at com.dotmarketing.cms.urlmap.filters.URLMapFilter.doFilter(URLMapFilter.java:114) ~[dotcms_5.0.0_999999.jar:?]
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:193) ~[catalina.jar:8.5.32]
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166) ~[catalina.jar:8.5.32]
	at com.dotmarketing.filters.VanityURLFilter.doFilter(VanityURLFilter.java:104) ~[dotcms_5.0.0_999999.jar:?]
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:193) ~[catalina.jar:8.5.32]
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166) ~[catalina.jar:8.5.32]
	at org.tuckey.web.filters.urlrewrite.UrlRewriteFilter.doFilter(UrlRewriteFilter.java:399) ~[urlrewritefilter-4.0.4.jar:4.0.4]
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:193) ~[catalina.jar:8.5.32]
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166) ~[catalina.jar:8.5.32]
	at com.dotmarketing.filters.TimeMachineFilter.doFilter(TimeMachineFilter.java:134) ~[dotcms_5.0.0_999999.jar:?]
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:193) ~[catalina.jar:8.5.32]
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166) ~[catalina.jar:8.5.32]
	at com.dotmarketing.filters.ThreadNameFilter.doFilter(ThreadNameFilter.java:88) ~[dotcms_5.0.0_999999.jar:?]
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:193) ~[catalina.jar:8.5.32]
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166) ~[catalina.jar:8.5.32]
	at com.dotmarketing.filters.CookiesFilter.doFilter(CookiesFilter.java:46) ~[dotcms_5.0.0_999999.jar:?]
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:193) ~[catalina.jar:8.5.32]
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166) ~[catalina.jar:8.5.32]
	at com.dotmarketing.filters.CharsetEncodingFilter.doFilter(CharsetEncodingFilter.java:102) ~[dotcms_5.0.0_999999.jar:?]
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:193) ~[catalina.jar:8.5.32]
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166) ~[catalina.jar:8.5.32]
	at com.dotcms.filters.interceptor.AbstractWebInterceptorSupportFilter.doFilter(AbstractWebInterceptorSupportFilter.java:90) ~[dotcms_5.0.0_999999.jar:?]
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:193) ~[catalina.jar:8.5.32]
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166) ~[catalina.jar:8.5.32]
	at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:198) ~[catalina.jar:8.5.32]
	at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:96) ~[catalina.jar:8.5.32]
	at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:493) ~[catalina.jar:8.5.32]
	at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:140) ~[catalina.jar:8.5.32]
	at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:81) ~[catalina.jar:8.5.32]
	at org.apache.catalina.valves.AbstractAccessLogValve.invoke(AbstractAccessLogValve.java:650) ~[catalina.jar:8.5.32]
	at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:87) ~[catalina.jar:8.5.32]
	at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:342) ~[catalina.jar:8.5.32]
	at org.apache.coyote.http11.Http11Processor.service(Http11Processor.java:800) ~[tomcat-coyote.jar:8.5.32]
	at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:66) ~[tomcat-coyote.jar:8.5.32]
	at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:800) ~[tomcat-coyote.jar:8.5.32]
	at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1471) ~[tomcat-coyote.jar:8.5.32]
	at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:49) ~[tomcat-coyote.jar:8.5.32]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149) [?:1.8.0_181]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624) [?:1.8.0_181]
	at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61) ~[tomcat-util.jar:8.5.32]
	at java.lang.Thread.run(Thread.java:748) [?:1.8.0_181]
Caused by: com.dotmarketing.portlets.workflows.model.WorkflowActionFailureException: Contentlet [53be627a-9864-4803-8cbd-56349adb8197] has invalid / missing field(s).
	at com.dotmarketing.portlets.workflows.actionlet.SaveContentActionlet.executeAction_aroundBody0(SaveContentActionlet.java:85) ~[dotcms_5.0.0_999999.jar:?]
	at com.dotmarketing.portlets.workflows.actionlet.SaveContentActionlet$AjcClosure1.run(SaveContentActionlet.java:1) ~[dotcms_5.0.0_999999.jar:?]
	at org.aspectj.runtime.reflect.JoinPointImpl.proceed(JoinPointImpl.java:149) ~[aspectjrt-1.8.10.jar:?]
	at com.dotcms.aspects.aspectj.AspectJDelegateMethodInvocation.proceed(AspectJDelegateMethodInvocation.java:43) ~[dotcms_5.0.0_999999.jar:?]
	at com.dotmarketing.db.LocalTransaction.wrapReturnWithListeners(LocalTransaction.java:40) ~[dotcms_5.0.0_999999.jar:?]
	at com.dotcms.aspects.interceptors.WrapInTransactionMethodInterceptor.invoke(WrapInTransactionMethodInterceptor.java:23) ~[dotcms_5.0.0_999999.jar:?]
	at com.dotcms.aspects.aspectj.WrapInTransactionAspect.invoke(WrapInTransactionAspect.java:41) ~[dotcms_5.0.0_999999.jar:?]
	at com.dotmarketing.portlets.workflows.actionlet.SaveContentActionlet.executeAction(SaveContentActionlet.java:50) ~[dotcms_5.0.0_999999.jar:?]
	at com.dotmarketing.portlets.workflows.business.WorkflowAPIImpl.fireWorkflowPostCheckin_aroundBody148(WorkflowAPIImpl.java:2000) ~[dotcms_5.0.0_999999.jar:?]
	... 86 more
Caused by: com.dotmarketing.portlets.contentlet.business.DotContentletValidationException: Contentlet [53be627a-9864-4803-8cbd-56349adb8197] has invalid / missing field(s).
	at com.dotcms.content.elasticsearch.business.ESContentletAPIImpl.validateContentlet_aroundBody136(ESContentletAPIImpl.java:4495) ~[dotcms_5.0.0_999999.jar:?]
	at com.dotcms.content.elasticsearch.business.ESContentletAPIImpl$AjcClosure137.run(ESContentletAPIImpl.java:1) ~[dotcms_5.0.0_999999.jar:?]
	at org.aspectj.runtime.reflect.JoinPointImpl.proceed(JoinPointImpl.java:149) ~[aspectjrt-1.8.10.jar:?]
	at com.dotcms.aspects.aspectj.AspectJDelegateMethodInvocation.proceed(AspectJDelegateMethodInvocation.java:43) ~[dotcms_5.0.0_999999.jar:?]
	at com.dotcms.aspects.interceptors.CloseDBIfOpenedMethodInterceptor.invoke(CloseDBIfOpenedMethodInterceptor.java:29) ~[dotcms_5.0.0_999999.jar:?]
	at com.dotcms.aspects.aspectj.CloseDBIfOpenedAspect.invoke(CloseDBIfOpenedAspect.java:41) ~[dotcms_5.0.0_999999.jar:?]
	at com.dotcms.content.elasticsearch.business.ESContentletAPIImpl.validateContentlet(ESContentletAPIImpl.java:4388) ~[dotcms_5.0.0_999999.jar:?]
	at com.dotcms.content.elasticsearch.business.ESContentletAPIImpl.validateContentlet_aroundBody140(ESContentletAPIImpl.java:4831) ~[dotcms_5.0.0_999999.jar:?]
	at com.dotcms.content.elasticsearch.business.ESContentletAPIImpl$AjcClosure141.run(ESContentletAPIImpl.java:1) ~[dotcms_5.0.0_999999.jar:?]
	at org.aspectj.runtime.reflect.JoinPointImpl.proceed(JoinPointImpl.java:149) ~[aspectjrt-1.8.10.jar:?]
	at com.dotcms.aspects.aspectj.AspectJDelegateMethodInvocation.proceed(AspectJDelegateMethodInvocation.java:43) ~[dotcms_5.0.0_999999.jar:?]
	at com.dotcms.aspects.interceptors.CloseDBIfOpenedMethodInterceptor.invoke(CloseDBIfOpenedMethodInterceptor.java:29) ~[dotcms_5.0.0_999999.jar:?]
	at com.dotcms.aspects.aspectj.CloseDBIfOpenedAspect.invoke(CloseDBIfOpenedAspect.java:41) ~[dotcms_5.0.0_999999.jar:?]
	at com.dotcms.content.elasticsearch.business.ESContentletAPIImpl.validateContentlet(ESContentletAPIImpl.java:4817) ~[dotcms_5.0.0_999999.jar:?]
	at com.dotcms.content.elasticsearch.business.ESContentletAPIImpl.internalCheckin(ESContentletAPIImpl.java:3047) ~[dotcms_5.0.0_999999.jar:?]
	at com.dotcms.content.elasticsearch.business.ESContentletAPIImpl.lambda$17(ESContentletAPIImpl.java:2924) ~[dotcms_5.0.0_999999.jar:?]
	at com.dotcms.concurrent.lock.StripedLockImpl.tryLock(StripedLockImpl.java:79) ~[dotcms_5.0.0_999999.jar:?]
	at com.dotcms.concurrent.lock.StripedLockImpl.tryLock(StripedLockImpl.java:47) ~[dotcms_5.0.0_999999.jar:?]
	at com.dotcms.concurrent.lock.IdentifierStripedLock.tryLock(IdentifierStripedLock.java:19) ~[dotcms_5.0.0_999999.jar:?]
	at com.dotcms.content.elasticsearch.business.ESContentletAPIImpl.checkin_aroundBody120(ESContentletAPIImpl.java:2923) ~[dotcms_5.0.0_999999.jar:?]
	at com.dotcms.content.elasticsearch.business.ESContentletAPIImpl$AjcClosure121.run(ESContentletAPIImpl.java:1) ~[dotcms_5.0.0_999999.jar:?]
	at org.aspectj.runtime.reflect.JoinPointImpl.proceed(JoinPointImpl.java:149) ~[aspectjrt-1.8.10.jar:?]
	at com.dotcms.aspects.aspectj.AspectJDelegateMethodInvocation.proceed(AspectJDelegateMethodInvocation.java:43) ~[dotcms_5.0.0_999999.jar:?]
	at com.dotmarketing.db.LocalTransaction.wrapReturnWithListeners(LocalTransaction.java:40) ~[dotcms_5.0.0_999999.jar:?]
	at com.dotcms.aspects.interceptors.WrapInTransactionMethodInterceptor.invoke(WrapInTransactionMethodInterceptor.java:23) ~[dotcms_5.0.0_999999.jar:?]
	at com.dotcms.aspects.aspectj.WrapInTransactionAspect.invoke(WrapInTransactionAspect.java:41) ~[dotcms_5.0.0_999999.jar:?]
	at com.dotcms.content.elasticsearch.business.ESContentletAPIImpl.checkin(ESContentletAPIImpl.java:2899) ~[dotcms_5.0.0_999999.jar:?]
	at com.dotcms.content.elasticsearch.business.ESContentletAPIImpl.checkin(ESContentletAPIImpl.java:6381) ~[dotcms_5.0.0_999999.jar:?]
	at com.dotcms.content.elasticsearch.business.ESContentletAPIImpl.checkin_aroundBody112(ESContentletAPIImpl.java:2820) ~[dotcms_5.0.0_999999.jar:?]
	at com.dotcms.content.elasticsearch.business.ESContentletAPIImpl$AjcClosure113.run(ESContentletAPIImpl.java:1) ~[dotcms_5.0.0_999999.jar:?]
	at org.aspectj.runtime.reflect.JoinPointImpl.proceed(JoinPointImpl.java:149) ~[aspectjrt-1.8.10.jar:?]
	at com.dotcms.aspects.aspectj.AspectJDelegateMethodInvocation.proceed(AspectJDelegateMethodInvocation.java:43) ~[dotcms_5.0.0_999999.jar:?]
	at com.dotcms.aspects.interceptors.CloseDBIfOpenedMethodInterceptor.invoke(CloseDBIfOpenedMethodInterceptor.java:29) ~[dotcms_5.0.0_999999.jar:?]
	at com.dotcms.aspects.aspectj.CloseDBIfOpenedAspect.invoke(CloseDBIfOpenedAspect.java:41) ~[dotcms_5.0.0_999999.jar:?]
	at com.dotcms.content.elasticsearch.business.ESContentletAPIImpl.checkin(ESContentletAPIImpl.java:2818) ~[dotcms_5.0.0_999999.jar:?]
	at com.dotmarketing.portlets.contentlet.business.ContentletAPIInterceptor.checkin(ContentletAPIInterceptor.java:256) ~[dotcms_5.0.0_999999.jar:?]
	at com.dotmarketing.portlets.workflows.actionlet.SaveContentActionlet.executeAction_aroundBody0(SaveContentActionlet.java:76) ~[dotcms_5.0.0_999999.jar:?]
	at com.dotmarketing.portlets.workflows.actionlet.SaveContentActionlet$AjcClosure1.run(SaveContentActionlet.java:1) ~[dotcms_5.0.0_999999.jar:?]
	at org.aspectj.runtime.reflect.JoinPointImpl.proceed(JoinPointImpl.java:149) ~[aspectjrt-1.8.10.jar:?]
	at com.dotcms.aspects.aspectj.AspectJDelegateMethodInvocation.proceed(AspectJDelegateMethodInvocation.java:43) ~[dotcms_5.0.0_999999.jar:?]
	at com.dotmarketing.db.LocalTransaction.wrapReturnWithListeners(LocalTransaction.java:40) ~[dotcms_5.0.0_999999.jar:?]
	at com.dotcms.aspects.interceptors.WrapInTransactionMethodInterceptor.invoke(WrapInTransactionMethodInterceptor.java:23) ~[dotcms_5.0.0_999999.jar:?]
	at com.dotcms.aspects.aspectj.WrapInTransactionAspect.invoke(WrapInTransactionAspect.java:41) ~[dotcms_5.0.0_999999.jar:?]
	at com.dotmarketing.portlets.workflows.actionlet.SaveContentActionlet.executeAction(SaveContentActionlet.java:50) ~[dotcms_5.0.0_999999.jar:?]
	at com.dotmarketing.portlets.workflows.business.WorkflowAPIImpl.fireWorkflowPostCheckin_aroundBody148(WorkflowAPIImpl.java:2000) ~[dotcms_5.0.0_999999.jar:?]
	... 86 more
```

## Context
This issue prevents the user from publishing the content from the workflow actions available on the Content Search portlet.
This was reported by client in this support issue: [97470](https://dotcms.zendesk.com/agent/tickets/97470)

## Your Environment
* dotCMS version used: 5.0.3
* Browser Name and version: Not relevant
* Operating System and version: Not relevant
* Application Server and version: Not relevant
* Java Brand and version: Not relevant
* Database and version: Not relevant
* Application module (if apply): Content management

**Comment by jgambarios** [comment]
PR: https://github.com/dotCMS/core/pull/16362

**Comment by bryanboza** [comment]
We are able to get the same case with the follow steps. 
1. Login as admin
2. Create a new content type or edit an existing one 
3. Add a field of tag type (DON'T set it as required)
4. Add some pieces of content leaving this field blank and just save it
5. Edit the content type and now set this field as required 
6. Right-click one of the created pieces of content and try to execute the publish action 

We need a better error handling here, also we need to improve the error message

**Comment by bryanboza** [comment]
Fixed, tested on master // Postgres // FF

**Comment by jgambarios** [comment]
PR: https://github.com/dotCMS/core/pull/16510

**Comment by fabrizzio-dotCMS** [comment]
I was able to break it. 
Create a CT with two fields 
1 Text field (nonrequired)  and 1 Tag field (required)

Create an instance of the new content-type from the content search. 
Pick the tag label 'healthcare'. 
Hit Save
Then select the new content instance. Open the Bulk Actions dialog and hit Publish.
The same error pops up from there.

This doesn't always happen with all tags. But I was able to reproduce it consistently using the tag 'healthcare'

![Screen Shot 2019-05-06 at 11 32 05 AM](https://user-images.githubusercontent.com/37148755/57243158-a1460480-6ff2-11e9-90a1-b29453c2324f.png)


![Screen Shot 2019-05-06 at 11 26 02 AM](https://user-images.githubusercontent.com/37148755/57243088-7491ed00-6ff2-11e9-947d-d6e07aa57ade.png)
